inyoka_theme_default/templates/ikhaya/includes/ikhaya-article-date.html

```diff
@@ -0,0 +1,24 @@
+{#
+  ikhaya/ikhaya-article-date.html
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
empty lines under it are removed, just this one is left to separate it from the license – GitHub can't know this :D